### DISCUSSION TITLE
Change step size to match pitch?

### DISCUSSION THREAD

**Discussion by mockendon** [kachurovskiy/nanoels#62] [Ideas]
I wonder if it would be possible to change the distance that the lead screw 'jogs' when the 'right' or 'left' button is pressed briefly.  The jog distance is always 1mm irrespective of the pitch set.  Would it be possible to jog by 1 x pitch?  I think this would be really helpful especially when setting the r-stop and l-stop for turning to a shoulder.

**Comment by kachurovskiy** [comment]
Matt, currently the logic is to jog 1mm or more while staying in sync: https://github.com/kachurovskiy/nanoels/blob/main/h1/NanoEls.ino#L608

You can change the 1mm constant by replacing `100.0` in lines `608`, `621` and `624` with a different value e.g. `25.0`.

There's a discussion around precision movements in https://github.com/kachurovskiy/nanoels/discussions/45#discussioncomment-2199039 - please feel free to comment there as well. Cheers!

**Reply by mockendon** [reply]
I realised that the steps could be bigger when cutting an imperial (24tpi) thread today actually.  It is the smaller steps that I'd like and your answer is perfect even half mm will really help me.  Thank you!
